code/database/migrations/2022_08_13_223049_update_users_model.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up():void
```

Comment:
dale un espacio entre : y void
```suggestion
    public function up(): void
```